lapce-ui/src/editor/view.rs

```diff
@@ -465,8 +465,6 @@ impl LapceEditorView {
     fn cursor_region(data: &LapceEditorBufferData, text: &mut PietText) -> Rect {
         let offset = data.editor.cursor.offset();
         let (line, col) = data.doc.buffer().offset_to_line_col(offset);
-        let inlay_hints = data.doc.line_phantom_text(&data.config, line);
-        let col = inlay_hints.col_at(col);
```

Comment:
You probably need to change this to ```let col = inlay_hints.col_after(col, false);```

Reply:
yes,I'm not check the `inlay_hints` code. I had change `col_at` to `col_after`.This is more suitable.And it work well。